app/assets/javascripts/govuk-admin-template/govuk-admin.js

```diff
@@ -85,16 +85,23 @@
 
   // Google Analytics event tracking
   // Label and value are optional
-  GOVUKAdmin.trackEvent = function(action, label, value) {
-
+  GOVUKAdmin.trackEvent = function(action, label, value, category) {
```

Comment:
I wonder if this is the right time to make a breaking change to this API and switch to a more flexible method signature: eg `GOVUKAdmin.trackEvent(options)`

Category is typically the first parameter, and if using the Google docs as reference is required. The default we use isn't useful.